### DISCUSSION TITLE
can snackbar show in top of window?

### DISCUSSION THREAD

**Discussion by li-zheng-hao** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2768] [Q&A] [answered]
In the demo, the snackbar appears from the bottom of the window

**Comment by Keboo** [comment, marked as answer]
Yes, this is possible. The location where the `Snackbar` appears is determined by where you place the `Snackbar` control within your XAML. 

For example, if the root control in your Window is a `DockPanel`, something like this will cause the `Snackbar` to be placed at the top.
```xaml
<Window>
  <DockPanel>
    <materialDesign:Snackbar
                        MessageQueue="{materialDesign:MessageQueue}"
                        DockPanel.Dock="Top"/>
  </DockPanel>
</Window>
```

**Reply by li-zheng-hao** [reply]
thanks for your reply！
